lib/shipit/webhooks.rb

```diff
@@ -0,0 +1,32 @@
+require 'shipit/webhooks/handlers'
+
+module Shipit
+  module Webhooks
+    def self.default_handlers
+      {
+        'push' => [Handlers::PushHandler],
+        'status' => [Handlers::StatusHandler],
+        'membership' => [Handlers::MembershipHandler],
+        'check_suite' => [Handlers::CheckSuiteHandler],
+      }
+    end
+
+    def self.handlers
+      @handlers ||= reset_handler_registry
+    end
+
+    def self.reset_handler_registry
```

Comment:
Couple style nitpicks here.

I tend to prefer to reopen the singleton class (`class << self`) rather than use `def self.method`, simply because it's much easier for full text search. Here since it's purely a static module, you can also use `extend self`, some people frown upon it, but I IMO it's fine.

Second is naming, I think `reset_handlers!`, would be a better name.

Reply:
Addressed by e90f024 and baa9422  respectively.